changelog/v1.14.0-beta10/listener-connection-config.yaml

```
@@ -0,0 +1,6 @@
+changelog:
+  - type: NEW_FEATURE
+    issueLink: https://github.com/solo-io/gloo/issues/7149
```

since we'll need a release before closing
```suggestion
    issueLink: https://github.com/solo-io/gloo/issues/7149
    resolvesIssue: false
```